tests/aws/services/stepfunctions/v2/services/test_lambda_task.snapshot.json

```diff
@@ -549,7 +549,7 @@
       }
     }
   },
-  "tests/aws/services/stepfunctions/v2/services/test_lambda_task.py::TestTaskLambda::test_invoke_json_values[00]": {
```

Comment:
Do these changes need to remain here or is the extra PR sufficient?

Reply:
I don't know what you mean with an extra PR?
The change here is necessary because pytest 8 changed the way paramtrized tests are named.
This change is definitely small enough to have in here and avoid the additional overhead of creating an explicit PR handling this upgrade, if this is what you mean.